compose/foundation/foundation/src/jsWasmMain/kotlin/androidx/compose/foundation/text/KeyEventHelpers.js.kt

```diff
@@ -20,4 +20,8 @@ import androidx.compose.ui.input.key.KeyEvent
 
 internal actual fun KeyEvent.cancelsTextSelection(): Boolean = false
 
-internal actual fun showCharacterPalette():Unit = TODO("implement native showCharacterPalette")
+internal actual fun showCharacterPalette() {
+    // There is no browser API to make such a system call neither directly nor indirectly:
+    // NSApplication* app = [NSApplication sharedApplication];
+    // [app orderFrontCharacterPalette:nil];
```

Comment:
Maybe this link is worth mentioning - https://github.com/whatwg/html/issues/8358